src/app/views/tour/Tour.tsx

```diff
@@ -6,21 +6,23 @@ import { IRootState } from '../../../types/root';
 
 import { toggleTourState, setNextTourStep } from '../../services/actions/tour-action-creator';
 import { TourTip } from './custom-components/Tourtip';
-import { BEGINNER_TOUR, ADVANCED_TOUR } from './utils/steps'
+import { BEGINNER_TOUR } from './utils/steps'
 import { useDispatch, useSelector } from 'react-redux';
 import { translateMessage } from '../../utils/translate-messages';
 import { FormattedMessage } from 'react-intl';
 import { setSampleQuery } from '../../services/actions/query-input-action-creators';
 import { componentNames, eventTypes, telemetry } from '../../../telemetry';
 import { geLocale } from '../../../appLocale';
+import { getTourSteps } from './utils/blob-steps/getSteps';
+import { setTimeout } from 'timers';
 
 const GETour = () => {
 
   const { tour } = useSelector((state: IRootState) => state);
-  const {step, beginner, isRunning} = tour;
+  const { step, beginner, isRunning, tourSteps, pending } = tour;
   const dispatch = useDispatch();
   const [run, setRun] = useState(isRunning);
-  const [steps] = useState(beginner === true ? BEGINNER_TOUR : ADVANCED_TOUR);
+  const [steps, setSteps] = useState(BEGINNER_TOUR);
```

Comment:
*Question*: This `BEGINNER_TOUR` is a constant set to an empty array. Is there a reason why we shouldn't just pass an empty array instead of the constant?

Reply:
The setSteps function is expecting an array of type ITourSteps[]. So even if the array is empty, it still has to be of type ITourSteps[]